3 - Assignment/240409 - Rust Basic + Roguelike Game, Week 2/elevator-events/src/lib.rs

```diff
@@ -1,7 +1,12 @@
 #[derive(Debug)]
 // An event in the elevator system that the controller must react to.
 enum Event {
-    // TODO: add required variants
+    ButtonPressed(Box<Event>),
+    CarDoorClosed,
+    CarDoorOpened,
+    LobbyCall(Direction,i32),
+    CarFloor(i32),
```

Comment:
1. `cargo fmt`를 통해 코드 스타일을 맞춰주세요.
2. `LobbyCall`이나 `CarFloor`는 `Event`라는 타입의 성격과는 맞지 않아 보입니다. 차라리 `Button`이라는 열거체를 새로 하나 만드는 게 어떨까요?
```suggestion
    LobbyCall(Direction, i32),
    CarFloor(i32),
```